doc/API.md

```diff
@@ -35,14 +35,15 @@ Display general information about the current daemon state.
 
 #### Response
 
-| Field                | Type    | Description                                                     |
-| -------------------- | ------- | --------------------------------------------------------------- |
-| `blockheight`        | integer | Current block height                                            |
-| `network`            | string  | Answer can be `mainnet`, `testnet`, `regtest`                   |
-| `sync`               | float   | The synchronization progress as percentage (`0 < sync < 1`)     |
-| `version`            | string  | Version following the [SimVer](http://www.simver.org/) format   |
-| `vaults`             | integer | Current number of vaults (unconfirmed are included)             |
-| `managers_threshold` | integer | Number of managers needed for spending the `unvault_tx`         |
+| Field                | Type         | Description                                                            |
+| -------------------- | ------------ | ---------------------------------------------------------------------- |
+| `blockheight`        | integer      | Current block height                                                   |
+| `network`            | string       | Answer can be `mainnet`, `testnet`, `regtest`                          |
+| `sync`               | float        | The synchronization progress as percentage (`0 < sync < 1`)            |
+| `version`            | string       | Version following the [SimVer](http://www.simver.org/) format          |
+| `vaults`             | integer      | Current number of vaults (unconfirmed are included)                    |
+| `managers_threshold` | integer      | Number of managers needed for spending the `unvault_tx`                |
+| `descriptors`        | string array | `deposit_descriptor`, `unvault_descriptor` & `cpfp_descriptor`         |
```

Comment:
It's not a string array, it's an object (`{}`, not `[]`).

How about something like:
```
 | descriptors        | object       |  Three `string` entries: `deposit`, `unvault` and `cpfp` for the three Miniscript descriptors |
 ```

Reply:
Pushed the required changes